v1/lesley-service.js

```diff
@@ -8,7 +8,21 @@ module.exports = function (RED) {
     this.options = {};
     this.options.name = config.name;
     this.options.url = config.url;
-    this.service = new restAPI.Service({ host: this.options.url });
+    this.options.notificationMethod = config.notificationMethod;
+    this.options.methodValue = config.methodValue;
+    if (this.options.notificationMethod === 'callback') {
```

Comment:
Šitoj vietoj teoriškai gali būti neinicializuotas service. Geriau būtų sukurti default config, tuomet ifuose pakeisti kuriuos reikia laukus ir galiausiai sukurti servisą. Galbūt tam galima panaudot `this.options` jei jis niekur kitur kitaip nenaudojamas.